tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -51,12 +53,25 @@ def setUp(self):
         """Pretest settings."""
         self.ps = PastaSauce()
         self.desired_capabilities['name'] = self.id()
-        self.admin = Admin(
-            use_env_vars=True,
-            pasta_user=self.ps,
-            capabilities=self.desired_capabilities
-        )
-        self.admin.login()
+        if not LOCAL_RUN:
+            self.admin = Admin(
+                use_env_vars=True,
+                pasta_user=self.ps,
+                capabilities=self.desired_capabilities
+            )
+            self.content = ContentQA(
+                use_env_vars=True,
+                pasta_user=self.ps,
+                capabilities=self.desired_capabilities
+            )
```

Comment:
Any subsequent user role needs to include `existing_driver` parameter; this is required for remote testing to keep from opening up more browser instances. The syntax for this case would be `existing_driver=self.admin.driver,`.